proxy/http2/Http2DependencyTree.h

```diff
@@ -37,80 +37,89 @@
 const static uint32_t K                               = 256;
 const static uint32_t HTTP2_DEPENDENCY_TREE_MAX_DEPTH = 256;
 
-template <typename T> class Http2DependencyTree
+namespace Http2DependencyTree
+{
+class Node
 {
 public:
-  class Node
+  Node(void *t = nullptr)
+    : active(false),
```

Comment:
As long as this is being changed, it should use class initializers. E.g., instead of initializing `active` and `queued` in two different constructors, change the member definitions.
```
bool active=false;
bool queued=false;
```